bench/binary_protocol.exs

```diff
@@ -32,14 +32,23 @@ defmodule BinaryProtocolBenchmark do
     {:ok, elixir_users: elixir_users, erlang_users: erlang_users}
   end
 
-  bench "erlang serialization" do
+  bench "erlang serialization (converted to binary)" do
     for user <- bench_context[:erlang_users] do
       serialize_user(user)
```

Comment:
Can you add `convert_to_binary: true` explicitly here? I think that would make this even clearer.